test/core/handshake/client_ssl.c

```diff
@@ -31,6 +31,11 @@
  *
  */
 
+#include "src/core/lib/iomgr/port.h"
+
+// This test won't work except with posix sockets enabled
```

Comment:
Right, there are several instances of this pattern, and that is the primary motivation for it. It's also a failsafe: if you ever try to run the test without posix sockets, it's guaranteed to fail, and opening the file immediately shows why.